examples/shapes_and_collections/hatches_styles.py

```diff
@@ -0,0 +1,47 @@
+"""
+===================
+Hatches Styles Demo
```

Comment:
```suggestion
Hatch style reference
```
- This is rather a reference than a demo.
- Not a native speaker, but I think "Hatches styles" is not grammatically correct.
- Capitalization style: We capitalize only the first letter of a heading (https://matplotlib.org/devdocs/devel/documenting_mpl.html#section-name-formatting).

Reply:
Changed filename to hatch_style_reference.py and updated the Heading.